content/react-part-2/reading/conditional-rendering/index.md

```diff
@@ -0,0 +1,452 @@
+---
+title: "Conditional Rendering"
+date: 2023-05-03T11:39:25-05:00
+draft: false
+weight: 2
+originalAuthor: Courtney Frey # to be set by page creator
+originalAuthorGitHub: speudusa # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+TODO: link to conditionals chapter in JS
+
+React components may need to display different things depending on certain conditions.  This is called **Conditional Rendering**. Conditional rendering allows you to change the output with logical operators.   This includes `if`, `if/else`, logical AND (`&&`) operator, and the ternary operator.
+
+## `if` statements
+
+We can use `if` statements to create conditions.  Those conditions then determine the output of our application.
+
+In the example below, we have two functions, `Rainy` and `Sunny`.  Each one returns a simple `<h1>` message.
+
+In the `App` function, we use an `if` statement and `props` to render which statement will appear in the browser.  The `if` block starts at line 15, and declares `isRainy` as `true`.  This will return the message from the `Rainy` function.  When `isRainy` is not `true`, it will return the ` <h1>No rain today!</h1>` message instead.
+
+{{% notice blue "Example Setting our Conditional to True" "rocket" %}}
+
+TODO: I've been using codesandbox for testing my examples -- would this be an option for students or too distracting?  the react.dev documentation uses codesandbox
```

Comment:
Suggestion -> make this a comment, so you can still find it but it doesn't appear as a line in the book.